proto/warden/warden/v1beta3/tx.proto

```diff
@@ -178,6 +179,7 @@ message MsgNewKeyRequest {
   uint64 keychain_id = 3;
   KeyType key_type = 4;
   uint64 rule_id = 5;
+  string max_fees = 6;
```

Comment:
wait, can you tell me more exactly why this can't be typed like this?

```proto
  repeated cosmos.base.v1beta1.Coin max_fees = 6 [
    (gogoproto.nullable)     = false,
    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
    (amino.dont_omitempty)   = true,
    (amino.encoding)         = "legacy_coins"
  ];
```

Reply:
Yes, this issue arises from the way we parse and execute these actions. I tried to use the `Coin` type, but `wardend` failed to start.

![image](https://github.com/user-attachments/assets/98511cc8-bda7-41fe-851d-6b642977b165)

![image](https://github.com/user-attachments/assets/d9fecabc-ddb4-4e65-9e28-3c766f8c33c5)

![image](https://github.com/user-attachments/assets/2862dafd-6168-4dbe-86bb-77b9e7ad9477)

Reply:
I hope this helps, but I'm happy to explain it further if needed